PopPUNK/assign.py

```diff
@@ -66,13 +68,20 @@ def assign_query(dbFuncs,
     from .utils import update_distance_matrices
     from .utils import createOverallLineage
 
+    from .web import sketch_to_hdf5
+    
     createDatabaseDir = dbFuncs['createDatabaseDir']
     constructDatabase = dbFuncs['constructDatabase']
     joinDBs = dbFuncs['joinDBs']
     queryDatabase = dbFuncs['queryDatabase']
     readDBParams = dbFuncs['readDBParams']
     getSeqsInDb = dbFuncs['getSeqsInDb']
 
+    # if running simple query
+    print_full_clustering = False
+    if update_db:
+        print_full_clustering = True
+
```

Comment:
This variable is no longer used, instead we set `printCSV` as `write_references or update_db` (see line 214). I think on lineage mode, this is always done anyway.

So this section can be removed entirely